terraform/environments/delius-jitbit/templates/task_definition.json

```diff
@@ -28,6 +28,22 @@
       {
         "name": "ConnectionStrings__DBConnectionString",
         "valueFrom": "${DATABASE_PASSWORD_CONNECTION_STRING_ARN}"
+      },
+      {
+        "name": "AttachmentsS3Login",
+        "valueFrom": "${S3_USER_KEY_ARN}"
+      },
+      {
+        "name": "AttachmentsS3Password",
+        "valueFrom": "${S3_USER_SECRET_ARN}"
```

Comment:
If the ECS task needs to communicate with S3, we should be looking at giving the task a role with the necessary permissions instead of giving it static credentials

Reply:
Perhaps this isn't possible with the current ECS module setup, but it will be after the improvements